jekyll/_cci2/server/v4.3/installation/phase-1-prerequisites.adoc

```diff
@@ -129,7 +129,7 @@ For high availability, and to avoid potential outages, you should provision subn
 | Kubernetes Version
 
 | 4.3.x
-| 1.22 - 1.24
+| 1.26 - 1.27
```

Comment:
hi folks, would this mean, 4.3.x requires k8s 1.26 minimally?
E.g., I shouldn't run 4.3.x on k8s 1.24?

If so, I think it would be good to mention a k8s version >=1.26 is required in https://circleci.com/docs/server/v4.3/installation/upgrade-server/#upgrade-steps 🤓